crates/precompiles/src/tip20/mod.rs

```diff
@@ -99,7 +99,6 @@ pub struct TIP20Token {
     opted_in_supply: u128,
     next_stream_id: u64,
     streams: Mapping<u64, RewardStream>,
```

Comment:
IIUC we should remove the entire `streams` mapping and `RewardStream` struct or am I overlooking something? Additinoally we no longer need `total_reward_per_second` or `last_update_time` esentially everything related to streaming rewards can be removed.